src/core_ocean/Registry.xml

```diff
@@ -713,6 +713,40 @@
 					possible_values="Any positive real number."
 		/>
 	</nml_record>
+	<nml_record name="time_varying_forcing" mode="init;forward">
+                <nml_option name="config_use_time_varying_atmospheric_forcing" type="logical" default_value=".false." units="unitless"
+                        description="If true calculate input forcing fields."
```

Comment:
Here and elsewhere, I noticed that spaces are being used instead of tabs, whereas all the whitespace at the beginning of each line is supposed to be from tabs.  Others may also not have been following this correctly but best to fix that for any additions from this PR.  I believe this is something the linter that @mark-petersen mentioned would catch.